src/component/auctions/auction.component.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+export const Auction = () => {
+  return(
+    <div>
+      
+    </div>
```

Comment:
Whats this?

Reply:
This will be a page which will contain all auctions